cms-contentful/it/com/commercetools/sunrise/cms/contentful/ContentfulCmsServiceIT.java

```diff
@@ -20,34 +21,12 @@
 public class ContentfulCmsServiceIT {
     private static final List<Locale> SUPPORTED_LOCALES = singletonList(Locale.GERMANY);
 
-    // credentials for contentful demo account
-    private static final String IT_PREFIX = "CONTENTFUL_";
-    private static final String IT_CF_SPACE_ID = IT_PREFIX + "SPACE_ID";
-    private static final String IT_CF_TOKEN = IT_PREFIX + "TOKEN";
```

Comment:
I would prefer to keep this constants because then you don't have to repeat the strings below.